src/set.js

```diff
@@ -0,0 +1,71 @@
+export default class Set {
+  constructor() {
+    this.data = [];
+    this.length = 0;
+  }
+
+  add( stuff ) {
+      if( this.data.length > 0 ){
+        for( var i = 0; i < this.data.length; i++ ){
+          if( this.data[i] == stuff ){
+            return "ERROR, " + stuff + " already exists"
+          }else{
+          return this.data.push(stuff)
+          }
+        }
+      }else{
+        return this.data.push(stuff)
+      }
+  }
+
+  isEmpty() {
+    if(this.data.length > 0) {
+    return false
```

Comment:
this is one of those cases where you can just do `this.data.length > 0`
that statement evaluates to true/false
I notice this pattern or look for a different solution every time i have an `if` that just returns `true` or `false`, that's a code smell for me.